### PR TITLE
sdk: generalize for any SDKs and enhance explanations

### PR DESCRIPTION
**Pull request by nicokosi**
1. There was a mix between 'gradle' and 'SDK'. We now only refer to 'SDK'.
Using the acronym for conciseness (the description explains the acronym).

2. Explanation for 'sdk use' was misleading. See https://sdkman.io/usage#use:

> It is important to realise that this will switch the candidate version for the current shell only.

3. Explanation for 'sdk current' has been enhanced, introducing the stable version concept aka 'candidate'.
See https://sdkman.io/usage#listcandidates:

> This will render a searchable alphabetic list with name, current stable default version (...)

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

~~**Version of the command being documented (if known):**~~

### CODE REVIEW

File: pages/common/sdk.md [hunk 1]
```diff
@@ -4,34 +4,34 @@
 > Supports Java, Groovy, Scala, Kotlin, Gradle, Maven, Vert.x and many others.
 > More information: <https://sdkman.io/usage>.
 
-- Install a specific version of Gradle:
+- Install a SDK version:
```
Comment (marchersimon) [inline pages/common/sdk.md:7]:
```suggestion
- Install an SDK version:
```
(https://oneminuteenglish.org/en/a-or-an-before-acronyms/)

But I think even better would be
```suggestion
- Install a specific SDK version:
```

File: pages/common/sdk.md [hunk 2]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk install {{gradle}} {{gradle_version}}`
+`sdk install {{sdk_name}} {{sdk_version}}`
 
-- Switch to a specific version of Gradle:
+- Use a specific SDK version in the current terminal:
```
Comment (marchersimon) [inline pages/common/sdk.md:11]:
```suggestion
- Use a specific SDK version for the current terminal session:
```

File: pages/common/sdk.md [hunk 3]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk use {{gradle}} {{gradle_version}}`
+`sdk use {{sdk_name}} {{sdk_version}}`
 
-- Check current Gradle version:
+- Show the stable version of a SDK:
```
Comment (marchersimon) [inline pages/common/sdk.md:15]:
```suggestion
- Show the stable version of the current SDK:
```
Not sure if this is correct

File: pages/common/sdk.md [hunk 4]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk current {{gradle}}`
+`sdk current {{sdk_name}}`
 
-- List all Software Development Kits available to install:
+- Show the stable versions of installed SDKs:
```
Comment (marchersimon) [inline pages/common/sdk.md:19]:
```suggestion
- Show the stable versions of all installed SDKs:
```

File: pages/common/sdk.md [hunk 5]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk list`
+`sdk current`
 
-- List all available versions for a specific Software Development Kit:
+- List all SDKs:
```
Comment (marchersimon) [inline pages/common/sdk.md:23]: Installed or available?

File: pages/common/sdk.md [hunk 6]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk list {{sdk_name}}`
+`sdk list`
 
-- List all installed Software Development Kits:
+- List all versions of a SDK:
```
Comment (marchersimon) [inline pages/common/sdk.md:27]: Again, installed or available? Also:
```suggestion
- List all versions of an SDK:
```

File: pages/common/sdk.md [hunk 7]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk current`
+`sdk list {{sdk_name}}`
 
-- Update Gradle to the latest version:
+- Upgrade a SDK to the latest stable version:
```
Comment (marchersimon) [inline pages/common/sdk.md:31]:
```suggestion
- Upgrade an SDK to the latest stable version:
```

File: pages/common/sdk.md [hunk 8]
```diff
@@ -4,34 +4,34 @@
...
 
-`sdk upgrade {{gradle}}`
+`sdk upgrade {{sdk_name}}`
 
-- Uninstall a particular version of Gradle:
+- Uninstall a SDK version:
```
Comment (marchersimon) [inline pages/common/sdk.md:35]:
```suggestion
- Uninstall a specific SDK version:
```

File: pages/common/sdk.md (hunk 3 above)
Comment (nicokosi) [reply pages/common/sdk.md:15]: I think it's not correct. :smile_cat:

Comment (marchersimon) [reply pages/common/sdk.md:15]: Then maybe
```suggestion
- Show the stable version of any available SDK:
```

File: pages/common/sdk.md (hunk 5 above)
Comment (marchersimon) [reply pages/common/sdk.md:23]:
```suggestion
- List all available SDKs:
```

**Review by marchersimon — APPROVED** [review]
👍🏽

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**